docs/source/support/release-notes.rst

```diff
@@ -13,6 +13,7 @@ HEAD — ongoing
 0.12.15 - 2019-05-06
 --------------------
 
+- DAML-LF Engine: if all labels in a record are set fields no longer need to be ordered
```

Comment:
Please create an issue and write motivation for this, and then reference the issue here.

Reply:
#988